### PR TITLE
11 Feb governance meeting report

### PR DESCRIPTION
**Pull request by valgrimm**

### CODE REVIEW

**Comment by XAMPPRocky** [comment]
@valgrimm This looks great! If you added a link to the zulip stream and mention the stream on Zulip is `#wg-governance`, so folks know where to go.

Stream link: https://rust-lang.zulipchat.com/#narrow/stream/223182-wg-governance

File: posts/inside-rust/2020-02-11-Goverance-wg [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Project groups
+
+You can find the [detailed minutes] on the [wg-governance] repository, but here is a quick summary: 
+
+* Project groups description RFC confirmed complete
+* Next items taken from Dec 3 backlog are access policy (NellShamrell is working on this),
+Discussion of where to discuss the pre-RFC process, and followup about moving (community, devtools, nursery) to rust-lang
+* XAMPPRocky is already working on a compliance bot to simplify administration
+* Timeslot adjusted to 19-20 CET / 1pm-2pm EST / 10-11am PST on Zulip, with same biweekly cadence.
+Zoom meetings will be used as needed, e.g. if a big RFC needs to be pushed through.
```
Comment (XAMPPRocky) [inline posts/inside-rust/2020-02-11-Goverance-wg]: Could you move these last two lines under "Next meeting", and mention that we're switching from video to text for meetings.

**Comment by lzutao** [comment]
Should rename to markdown i think

File: posts/inside-rust/2020-02-11-Goverance-wg (hunk 1 above)
Comment (valgrimm) [reply posts/inside-rust/2020-02-11-Goverance-wg]: Ok, will do.

**Comment by valgrimm** [comment]
Ok, adjusted the two things that needed fixed, thanks for the input.

**Pull request merged**